os.go

```diff
@@ -34,3 +35,80 @@ func mkfs(dev, fstype string) error {
 	}
 	return nil
 }
+
+func umount(mntpoint string) error {
+	tmpDir := os.TempDir()
+	var stderr bytes.Buffer
+
+	cmd := exec.Command(
+		"/bin/umount",
+		tmpDir,
+	)
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("umount stderr: %s", stderr.String())
+		return err
+	}
+	return nil
+}
+
+func tempMount(dev, fstype string, mountOptions ...string) error {
+	tmpDir := os.TempDir()
+	var stderr bytes.Buffer
+
+	mountArgs := []string{
+		"-t",
+		fstype,
+	}
+	mountArgs = append(mountArgs, mountOptions...)
+	mountArgs = append(mountArgs,
+		dev,
+		tmpDir,
+	)
+	cmd := exec.Command(
+		"/bin/mount",
+		mountArgs...,
+	)
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("mount stderr: %s", stderr.String())
+		return err
+	}
+	return nil
+}
```

Comment:
can we maybe use something like [github.com/moby/sys.Mount](https://pkg.go.dev/github.com/moby/sys@v0.0.0-20200315194828-07b9aaad3088/mount#Mount) and Umount here? I'm already a bit wary about using `exec.Command` for `mkfs` :sweat:

Reply:
sure, I will check what I can do. I mostly copied the style, thanks for the suggestion